### ISSUE TITLE
locpargen vs TGYRO give different results

### ISSUE THREAD

**Issue by smithsp** [gafusion/gacode#220]
Working with @avdeevag , we were trying to use OMFIT to run TGYRO on portal (PPPL) to get the input files for TGLF, but we ran across a problem: when we ran TGYRO it only produced inputs for the first radius (TGLF1 directory) - a problem that I may solicit the help of @jsachdev to solve.  So then I said to myself: @jcandy and @bellie created this great new option in `profiles_gen` - the `loc_rho` option - to get the local input files for TGLF or NEO.  So we tried to run that for `rho=0.01` - the case that had run successfully in TGYRO's TGLF1 directory (since we were using `TGYRO_USE_RHO=1` and `TGYRO_RMIN=0.01` in our `input.tgyro` file).  However, there were non-trivial differences in the files that were created, which prompted me to ask what's going on and open this issue.  The case is on `iris` in `/cluster-scratch/smithsp/test_input.tglf`.  In that directory is the starting `statefile.nc` and `gfile` (from TRXPL).  I then produced the `input.gacode` file with `module purge; module load atom/tglf_sat2; profiles_gen -i statefile.nc -g gfile`, and then the `input.tglf.locpargen` with `profiles_gen -i input.gacode -loc_rho 0.01`.  For the TGYRO based cases, I used OMFIT's TGLF_scan module, which sets up 99 radial points in TGYRO, and which I exported to `/cluster-scratch/smithsp/test_input.tglf/tgyro_dir` for inspection.  

TL;DR
The two files I expected to have identical results are
`/cluster-scratch/smithsp/test_input.tglf/input.tglf.locpargen` and `/cluster-scratch/smithsp/test_input.tglf/tgyro_dir/TGLF1/out.tglf.localdump`.
